app/javascript/locales/es.js

```diff
@@ -45,6 +45,9 @@ export default {
       quantity: 'Cantidad',
       measure: 'Unidad',
       selected: 'Ingredientes seleccionados',
+      default_unit: 'Unidad por defecto',
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'default_unit' is not in camel case      camelcase